3-platform/shopping-cart/db/postgres/01-create-shopping-cart.sql

```diff
@@ -0,0 +1,19 @@
+-- DROPPING TABLE IF EXISTS
+-- DROP TABLE IF EXISTS shopping_carts;
+
+-- CREATING TABLE "shopping_carts"
+CREATE TABLE IF NOT EXISTS shopping_carts
+(
+    id         VARCHAR(20) NOT NULL,
+    product_id VARCHAR(150)
```

Comment:
1. product_id can't be null. Add NOT NULL constraint.
2. add FK to products table.

Reply:
1 - Okay, it's done
2 - I can't because of "List productIds();"

Reply:
@sokimaaa , could you help us decide how should we store references to Product table?

Reply:
@sokimaaa, could you help us, please? Or, maybe, @YuraKhilko, we`ll resolve this issue mutually without outsiders? )

Reply:
shopping cart service should know nothing about Warehouse Database (e.g. Database per service Pattern). 

so, productId[] in this table is just like string[] or int[] without any relationships.

Reply:
@andrey-kiyas, make appropriate changes.

Reply:
And again we returned to where we started ))